packages/network-controller/src/NetworkController.ts

```diff
@@ -1827,6 +1838,16 @@ export class NetworkController extends BaseController<
     return this.state.networkConfigurationsByChainId[chainId];
   }
 
+  getNetworkClientIdByChainId(chainId: Hex): NetworkClientId | undefined {
```

Comment:
Is `findNetworkClientByChainId` not be sufficient for your use case? Or do you prefer this because it doesn't throw? I hesitate to introduce a similar sounding method because it could get confusing.

Reply:
Fixed here [ce6398f](https://github.com/MetaMask/core/pull/5622/commits/ce6398fcdafa47857849d72ca85e823d7cc11d2a) 🙏

Reply:
Thank you!